src/test/java/org/fusesource/jansi/AnsiRendererTest.java

```diff
@@ -107,9 +107,21 @@ public void testRenderInvalidMissingEnd() {
         assertEquals("@|bold foo", str);
     }
 
+    @Test
+    public void testRenderInvalidEndBeforeStart() {
+        try {
+            render("@|@");
```

Comment:
The JUnit5 syntax is now:
```
  assertThrows(IllegalArgumentException.class, () -> render("@|@"));
```
instead of the whole try/catch block.

Reply:
@gnodet Thanks for your reply. But as the project is set to JDK1.7 (https://github.com/fusesource/jansi/blob/master/pom.xml#L35-L36), I am not sure is the assertThrows using lambda expression can be used. Thats why I use the traditional try catch method.

Reply:
@gnodet Thanks for your reply. The main reason I use the traditional try catch block is because I see that the project is deployed with JDK1.7 (https://github.com/fusesource/jansi/blob/master/pom.xml#L35-L36) which lambda expression is not supported. But I agree that assertThrows from junit5 is a better approach. I have changed the code to use anonymous interface instance creation (instead of lambda expression) on assertThrows method.

Reply:
Right, I forgot about that.  Though it may be time to bump the requirements to Jdk 8 now...

Reply:
I've update the master branch to bump tests to JDK 8, so if you rebase/merge, you should be able to leverage that and use lambdas.

Reply:
Thanks. I have change the unit test and rebased the branches. It is working now.